form.html

```diff
@@ -0,0 +1,174 @@
+
+	  
+	  
+	
+    <form class="form-horizontal" accept-charset="UTF-8" action="https://usebasin.com/f/01873b5e2c07" method="POST">
+		 <div class="form-group">
+			<label class="control-label col-sm-12" for="name">Name</label>
+			<div class="col-sm-10">
+				<input type="text" class="form-control" name="name" id="name" placeholder="Name">
+			</div>
+    </div>
+    <br>		
+		<div class="form-group">
+		<label class="control-label col-sm-12" for="email-address">Email Address/Discord Username</label>
+			<div class="col-sm-10">
+				<input type="text" class="form-control" name="email-address" id="email-address" placeholder="Please let us know your preferred method of contact">
+			</div>
+    </div>
+    <br>		
+		<div class="form-group">
+			<label class="control-label col-sm-12" for="hear-about-wobbly">How did you hear about Wobbly?</label>
+			<div class="col-sm-10">
+				<input type="text" class="form-control" name="hear-about-wobbly" id="hear-about-wobbly" placeholder="How did you hear about Wobbly?">
+			</div>
+    </div>
+    <br>		
+		<div class="form-group">
+			<label class="control-label col-sm-12" for="contributing">How are you interested in contributing? (If you're a developer, please mention technical areas of expertise)</label>
+			<div class="col-sm-10">
+				<textarea rows="8" class="form-control"  name="contributing" id="contributing" placeholder="How are you interested in contributing?"></textarea>
+			</div>
+    </div>	
+    <br>		
+		<div class="form-group">
+			<label class="control-label col-sm-12" for="timezone">What timezone are you in?</label>
+			<div class="col-sm-10">
+				<input type="text" class="form-control" name="timezone" id="timezone" placeholder="Please use UTC time">
+			</div>
+    </div>	
+    <br>	
+		<div class="form-group">
+			<label class="control-label col-sm-12" for="amount-hours">Do you have a rough idea of how many hours/month you can contribute?</label>
+			<div class="col-sm-10">
+				<input type="text" class="form-control" name="amount-hours" id="amount-hours" placeholder="Do you have a rough idea of how many hours/month you can contribute?">
+			</div>
+    </div>
+    <br>		
+		<div class="form-group">
+			<label class="control-label col-sm-12" for="political-projects">Do you have any connections to other political projects? (optional)</label>
+			<div class="col-sm-10">
+				<input type="text" class="form-control" name="political-projects" id="political-projects" placeholder="If you would like to, please let us know of any other political projects you are involved with.">
+			</div>
+        </div>
+        <br>
+        <div class="form-group">
+            <div class="col-sm-offset-2 col-sm-12">
+              <button type="submit" class="btn btn-default">Submit</button>
+            </div>
+          </div>
+    </form>
+
```

Comment:
You can also delete all the lines below the `</form>` -- the `page` layout will add those automatically.